packages/x-charts-pro/src/FunnelChart/FunnelChart.test.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import { expect } from 'chai';
+import { createRenderer, describeConformance, screen } from '@mui/internal-test-utils';
+import { Unstable_FunnelChart as FunnelChart } from '@mui/x-charts-pro/FunnelChart';
+import { describeSkipIf, isJSDOM } from 'test/utils/skipIf';
+
+const config = {
+  series: [{ data: [{ value: 200 }, { value: 100 }] }],
+  margin: { top: 0, left: 0, bottom: 0, right: 0 },
+  width: 400,
+  height: 400,
+  hideLegend: true,
+} as const;
+
+describe('FunnelChart', () => {
+  const { render } = createRenderer();
+
+  describeConformance(
+    <FunnelChart height={100} width={100} series={[{ data: [{ value: 100 }, { value: 50 }] }]} />,
+    () => ({
+      classes: {} as any,
+      inheritComponent: 'svg',
+      render,
+      muiName: 'MuiFunnelChart',
+      testComponentPropWith: 'div',
+      refInstanceof: window.SVGSVGElement,
+      skip: [
+        'componentProp',
+        'componentsProp',
+        'slotPropsProp',
+        'slotPropsCallback',
+        'slotsProp',
+        'themeStyleOverrides',
+        'themeVariants',
+        'themeCustomPalette',
+        'themeDefaultProps',
+      ],
+    }),
+  );
+
+  it('should render "No data to display" when axes are empty arrays', () => {
+    render(<FunnelChart series={[]} width={100} height={100} />);
+
+    expect(screen.getByText('No data to display')).toBeVisible();
+  });
+
+  describeSkipIf(isJSDOM)('gap', () => {
+    it('should properly distance sections based on gap', async () => {
+      render(
+        <div
+          style={{
+            width: 400,
+            height: 400,
+          }}
+        >
+          <FunnelChart {...config} gap={13} />
+        </div>,
+      );
+      const paths = document.querySelectorAll<HTMLElement>('path.MuiFunnelSection-root');
+
+      const firstBCR = paths[0].getBoundingClientRect();
+      const secondBCR = paths[1].getBoundingClientRect();
+      expect(firstBCR.bottom).to.be.closeTo(
+        secondBCR.top - 13,
+        0.45,
```

Comment:
Why do we need such a big delta? Isn't 0.01 or 0.1 enough?

Reply:
The bigger the gap the bigger the difference.

The diff seems to scale as either the gap or the chart size grows.

It doesn't seem much affected by the division precision, and even when it is a "roundish" number, eg: `paddingInner(0.4)` it gives off a big difference. I suppose it might be something inside the d3scale padding.